apps/nextjs-pages-router/next.config.mjs

```diff
@@ -5,6 +5,10 @@ const withMakeswift = createWithMakeswift()
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  i18n: {
+    locales: ["en-US", "en-FR", "fr-FR"],
```

Comment:
Maybe it's just me but I don't want us to have `en-FR` in these examples. It's a weird quirk that BigCommerce uses but I wouldn't expect to be a common thing for people to do, specially because these locales aren't even meant to be public (i.e., they're only used "internally" and no visitor to the site should be able to see these locales).